tsfile/src/main/java/org/apache/iotdb/tsfile/read/TsFileSequenceReader.java

```diff
@@ -246,21 +246,21 @@ public Chunk readMemChunk(ChunkMetaData metaData) throws IOException {
   /**
    * not thread safe.
    *
-   * @param type -given tsfile data type
+   * @param type given tsfile data type
    */
   public PageHeader readPageHeader(TSDataType type) throws IOException {
-    return readPageHeader(type, -1);
+    return PageHeader.deserializeFrom(tsFileInput.wrapAsInputStream(), type);
```

Comment:
You should not modify here. because `readPageHeader(TSDataType type, long position)` still has the bug..
You should modify `readPageHeader(TSDataType type, long position)`, rather than this method.